openapi-issuer.yaml

```diff
@@ -108,6 +108,31 @@ paths:
           $ref: "#/components/responses/NotFound"
         default:
           $ref: "#/components/responses/Unexpected"
+    patch:
+      operationId: patchSignatureRequestStatus
+      summary: Change the status of a signature request
+      parameters:
+        - in: path
+          name: signatureRequestId
+          schema:
+            type: string
+            format: ulid
```

Comment:
is this format enfoced somewhat into the io-ts type?

Reply:
@lucacavallaro

Reply:
nope, right now we don't validate ULID at runtime